internal/compose/compose.go

```diff
@@ -199,20 +198,15 @@ func NewProject(name string, paths ...string) (*Project, error) {
 	// Passing a nil context here because we are on initialization.
 	ver, err := c.dockerComposeVersion(context.Background())
 	if err != nil {
-		logger.Errorf("Unable to determine Docker Compose version: %v. Defaulting to 1.x", err)
-		c.dockerComposeV1 = true
-		return &c, nil
+		return nil, fmt.Errorf("unable to determine Docker Compose version: %w", err)
```

Comment:
Making this an error now, the main reason for this to fail is that docker compose command is not found.